### PR TITLE
Fixes #23150 - Katello task page 404 error and refresh issue

### PR DESCRIPTION
**Pull request by sjha4**
### Creating this PR against 2 issues:
1. http://projects.theforeman.org/issues/23150
2. http://projects.theforeman.org/issues/23264
****
#### Steps to reproduce [23150](http://projects.theforeman.org/issues/23150)
1. Go to any Katello Resource > Task page (ex: Product>task , CV> Task etc.)
2. A 404 error is reported in the logs due to a call made to endpoint /katello/api/v2/tasks which doesn't exist.
****
#### Steps to reproduce [23264](http://projects.theforeman.org/issues/23264)
1. Go to any katello resource > Task [Ex: Product > Tasks]
2. Do a browser refresh
3. All tasks (returned by foreman_tasks bulk search) are displayed.

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/task.factory.js [hunk 1]
```diff
@@ -24,16 +24,14 @@ angular.module('Bastion.tasks').factory('Task',
...
-
-        var foremanTasksResource = BastionResource('foreman_tasks/api/tasks/:id/:action',
+        var resource = BastionResource('foreman_tasks/api/tasks/:id/:action',
             {},
             {
                 bulkSearch: {method: 'POST', isArray: true, params: { action: 'bulk_search'}}
             }
         );
 
+
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/task.factory.js]: What's this newline for? :)

Comment (sjha4) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/task.factory.js]: A double tap on Enter :grin:

**Comment by jlsherrill** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
@jturel : This should be lint free..:smile:

**Comment by bbuckingham** [comment]
Thanks @sjha4 !  Merging!

**Pull request merged**